chain/client/src/client.rs

```diff
@@ -626,8 +627,8 @@ impl Client {
         }
 
         let new_chunks = self.get_chunk_headers_ready_for_inclusion(&epoch_id, &prev_hash);
-        debug!(target: "client", "{:?} Producing block at height {}, parent {} @ {}, {} new chunks", validator_signer.validator_id(),
-               next_height, prev.height(), format_hash(head.last_block_hash), new_chunks.len());
+        debug!(target: "client", "{:?} Producing block at height {}, parent {} @ {}, {} new chunks: {:?}", validator_signer.validator_id(),
+               next_height, prev.height(), format_hash(head.last_block_hash), new_chunks.len(), new_chunks.keys().sorted());
```

Comment:
not relevant to the rest of the PR, but useful debug info

Reply:
Please make it friendly to grafana tempo:
`debug!(target: "client", next_height, prev_height = prev.height(), parent, ..., "Producing block");`